js/packages/web/src/components/CurrentUserBadge/index.tsx

```diff
@@ -158,7 +158,7 @@ export const CurrentUserBadge = (props: {
                 <h5>BALANCE</h5>
                 <Space direction="horizontal">
                   <SolCircle />
-                  <span>{formatNumber.format(balance)} SOL</span>
+                  <span>{formatNumber.format(balance)}</span>
```

Comment:
Should show the quote MINT here, let user know what they are missing.

Reply:
Agree